Gemfile

```diff
@@ -33,6 +33,8 @@ group :development, :test do
 
   gem 'puma'
 
+  gem 'haml'
```

Comment:
I was getting the same error locally when I set up the project for the first time. I think this was simply missed somehow?

https://circleci.com/gh/standardnotes/web/3

```
Running "haml:dist" (haml) task
Warning: You need to have Ruby and Haml installed and in your PATH for
this task to work. More info:
https://github.com/jhchen/grunt-haml2html Use --force to continue.
```